bigtable/admin.go

```diff
@@ -502,6 +519,24 @@ func (ac *AdminClient) DropAllRows(ctx context.Context, table string) error {
 // might be changed in backward-incompatible ways and is not recommended for
 // production use. It is not subject to any SLA or deprecation policy.
 func (ac *AdminClient) CreateTableFromSnapshot(ctx context.Context, table, cluster, snapshot string) error {
+	op, err := ac.CreateTableFromSnapshotAsync(ctx, table, cluster, snapshot)
+	if err != nil {
+		return err
+	}
+	ctx = mergeOutgoingMetadata(ctx, ac.md)
+	resp := btapb.Table{}
+	return AwaitOperation(ctx, op, &resp, ac.defaultInterval)
+}
+
+// CreateTableFromSnapshotAsync creates a table from snapshot.
+// The table will be created in the same cluster as the snapshot.
+// It returns an awaitable operation.
+//
+// This is a private alpha release of Cloud Bigtable snapshots. This feature
+// is not currently available to most Cloud Bigtable customers. This feature
+// might be changed in backward-incompatible ways and is not recommended for
+// production use. It is not subject to any SLA or deprecation policy.
+func (ac *AdminClient) CreateTableFromSnapshotAsync(ctx context.Context, table, cluster, snapshot string) (*longrunning.Operation, error) {
```

Comment:
PR does not mention adding these extra methods. Also seeing the work Async in a method name does not seem idiomatic. 

There have also been talks of generating a GAPIC admin surface for this API. I wonder if that is a better route forward for exposing these operations.

Reply:
You are right, added to PR comment. These were discussed after initial version of the PR was created.
I think Async is a good name. It means that you can call this function and it will be done on it's own time, and you don't have to wait for it.